controllers/azuremachine_tags.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2019 The Kubernetes Authors.
```

Reply:
hmmm there's a mix of 2018/2019 in the repo, what's the desired one @justaugustus ? We should probably update all of them (in a separate PR)